### PR TITLE
More customizations for toolbar and header

### PR DESCRIPTION
**Pull request by wkethman**
- Add ability to customize toolbar display - show/hide locate and start action buttons
- Add configurable action button to header
- These options accommodate Wyze Vacuum control

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 node_modules
 dist
-yarn.lock
\ No newline at end of file
+yarn.lock
+.DS_Store
```
Comment (denysdovhan) [inline .gitignore:4]: This should be in the global `.gitignore` file on your machine.

File: src/vacuum-card.js [hunk 2]
```diff
@@ -194,6 +218,25 @@ class VacuumCard extends LitElement {
...
+    }
+
+    const { header_action = [] } = this.config;
+
+    const execute = () => {
+      const [domain, name] = header_action.service.split('.');
+      this.hass.callService(domain, name, header_action.service_data);
+    };
+
+    return html`<div class="header_action"><ha-icon-button
```
Comment (denysdovhan) [inline src/vacuum-card.js:233]: `ha-icon-button` is changed in the latest HA. Please define icons with `ha-icon`.

File: src/vacuum-card.js [hunk 3]
```diff
@@ -448,18 +499,16 @@ class VacuumCard extends LitElement {
...
           <div class="header">
+            ${this.renderHeaderAction()}
             <div class="source">
               ${this.renderSource()}
             </div>
-            <div class="battery">
+            <div class="battery" @click="${() => this.handleMore()}" ?more-info="true">
               ${battery_level}% <ha-icon icon="${battery_icon}"></ha-icon>
             </div>
+
```
Comment (denysdovhan) [inline src/vacuum-card.js:511]: No empty line here.

File: src/vacuum-card.js [hunk 4]
```diff
@@ -403,21 +446,29 @@ class VacuumCard extends LitElement {
...
-            >
-            </ha-icon-button>
-
-            <ha-icon-button
-              icon="mdi:map-marker"
-              title="${localize('common.locate')}"
-              @click="${() => this.callService('locate', false)}"
-            >
-            </ha-icon-button>
+            ${ this.showStart == true ? startButton : ''}
```
Comment (denysdovhan) [inline src/vacuum-card.js:469]:
```suggestion
            ${this.showStart == true ? startButton : ''}
```

File: src/vacuum-card.js [hunk 5]
```diff
@@ -403,21 +446,29 @@ class VacuumCard extends LitElement {
...
+
+            ${ this.showLocate == true ? locateButton : ''}
```
Comment (denysdovhan) [inline src/vacuum-card.js:471]:
```suggestion
            ${this.showLocate == true ? locateButton : ''}
```

**Review by denysdovhan — CHANGES_REQUESTED** [review]

File: src/vacuum-card.js (hunk 2 above)
Comment (tzagim) [reply src/vacuum-card.js:233]: Hi @denysdovhan.
You don't intend to fit the card to the latest version of HA?

**Comment by denysdovhan** [comment]
Outdated. No recent activity.

**Pull request closed without merging**